roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -36,6 +39,14 @@ class RooFuncWrapper final : public RooAbsReal {
 
    void getGradient(double *out) const;
 
+   unsigned int NDim() const { return _params.size(); }
```

Comment:
Is that member function really needed? If yes, please rename it such that it follows RooFits `camelCase` convention, e.g. `nDim()` or maybe even better `nParams()` to be more self explanatory (otherwise it's not clear if this is the dimension of parameters, observables, or both).

Reply:
No it's not needed, I just kept the function from the previous design. I can remove it